page-objects/pages/TASKListPage.js

```diff
@@ -16,6 +18,9 @@ const taskTitleContainer = '.assignment.card .title';
 const taskDescriptionContainer = '.assignment .text-muted.ckcontent';
 const taskContainer = '.homework li.card';
 const deleteTaskButtonInPopup = '.delete-modal button.btn-submit';
+const clickWithoutDueDate = '[href="/homework/5fa3b9bea9c31a26f4d1da93"]';
+const ungradedTaskSection = '.v-window-item--active > section > .v-item-group > [aria-expanded="false"] > .v-expansion-panel-header';
+const ungradedTask = '.v-list-item__title';
 
```

Comment:
const ungradedTaskSection and const ungradedTask are bad selectors.
Ask your team, or we can try it ourselfs, to create data-testIds for those panels.